wezterm-gui/src/termwindow/mod.rs

```diff
@@ -2716,6 +2716,11 @@ impl TermWindow {
                     w.hide();
                 }
             }
+            HideOtherApplications => {
+                if let Some(w) = window.as_ref() {
+                    w.hide_other_apps();
+                }
```

Comment:
Should probably do this like the `HideApplication` action down on line 2751 (please move this alongside it)

```suggestion
                let con = Connection::get().expect("call on gui thread");
                con.hide_other_applications();
```

Reply:
Thanks, your suggested approach makes more sense.